regression/goto-bmc/goto-bmc-non-symex-ready-goto/negative1/main.desc

```diff
@@ -1,7 +1,7 @@
 CORE
 main.c
 
-^EXIT=[127|134]$
```

Comment:
:pick: The commit title refers to moving tests, but this part is changing the test.